src/main/java/seedu/exercise/logic/commands/CommandResult.java

```diff
@@ -21,6 +21,14 @@
      */
     private final boolean exit;
 
+    private boolean showExerciseList = false;
+
+    private boolean showRegimeList = false;
+
+    private boolean showScheduleList = false;
+
+    private boolean showSuggestionlist = false;
+
```

Comment:
Since only 1 list could be shown at any point in time, you may consider using integers to indicate which list is being shown. 
public static final int TYPE_EXERCISE_LIST = 0
public static final int TYPE_REGIME_LIST = 1
public static final int TYPE_SCHEDULE_LIST = 2
public static final int TYPE_SUGGEST_LIST = 3
private int currentListType;

Then for each of the CommandResult returned in the commands, you can just setList(TYPE_EXERCISE_LIST) instead of having 4 (or possibly more in future) different methods

Reply:
ok